### ISSUE TITLE
Pre-configured replies on several accounts

### ISSUE THREAD

**Issue by gshep** [gear-tech/gear#395]
### Problem to Solve

It is useful to emulate real users in gtest.

One has to implement a corresponding program to emulate user-behavior at the moment. For example, `test_bob` in #388 .

### Possible Solution

1. hard-code in gtest binary - easy but not flexible
2. describe 'request-reply' messaging in yaml-file and specify it as a program in test spec. For example:
```
programs:
  - id: 1
    path: target/wasm32-unknown-unknown/release/demo_signed_message.wasm
    init_message:
      # bob pubkey
      kind: bytes
      value: "8eaf04151687736326c9fea17e25fc5287613693c912909cb226aa4794f26a48"
  - id: 2
    path: test_bob.yaml
```

**Comment by gshep** [comment]
seems to be solved by 9b568e4333bb23f046b4ad66f38b38bfbe700c99

**Comment by NikVolf** [comment]
> seems to be solved by [9b568e4](https://github.com/gear-tech/gear/commit/9b568e4333bb23f046b4ad66f38b38bfbe700c99)

yep nice work
